techniques/failures/F94.html

```diff
@@ -59,7 +57,7 @@ <h3>Procedure</h3>
 			<section class="test-results">
 				<h3>Expected Results</h3>
 				<ul>
-					<li>If step #5 is false, then this failure condition applies and the content fails Success Criteria 1.4.4, Resize Text.</li>
+					<li>If step #4 is false, then this failure condition applies and the content fails Success Criteria 1.4.4, Resize Text.</li>
```

Comment:
I suggest "Success Criterion 1.4.4 Resize Text." (singular and without comma)

Reply:
```suggestion
					<li>If step #4 is false, then this failure condition applies and the content fails Success Criterion 1.4.4 Resize Text.</li>
```